src/main/scala/io/keen/client/scala/Logger.scala

```diff
@@ -0,0 +1,56 @@
+package io.keen.client.scala
+
+import grizzled.slf4j.Logging
+
+object Logger extends Logging {
```

Comment:
Totally :+1: on extracting consistent logging, but it would be nice not to make it a singleton that's hard to configure. Logging is something that ops teams always want to tweak for their environment—could we keep this compatible with the SLF4J interface, with a default instance throughout the package that uses `stdout` perhaps?

This discussion could be tabled to a subsequent pull request since it's a bit out-of-scope for the main subject at hand.